neuromation/cli/job.py

```diff
@@ -978,7 +978,15 @@ def format_fail(job: str, reason: Exception) -> str:
                         f"{STORAGE_MOUNTPOINT}/neuromation:ro"
                     )
                 )
-                # TODO (artem) print deprecation warning (issue #1009)
+                click.echo(
+                    click.style(
+                        f"DeprecationWarning: Option `--volume=HOME` is deprecated. "
+                        f"Use `--volume=ALL`.  Mountpoint will be available in "
+                        f"container via variable {NEUROMATION_HOME_ENV_VAR}",
```

Comment:
Could you please substitute `NEUROMATION_HOME_ENV_VAR` here?
Every time when I read texts like this I need to recall what the exact value of the mentioned constant is.
It reduces the code readability a little.
The constant has zero chance to be changed; in this case, we need more actions that just the renaming anyway.